projects/plugins/crm/src/rest-api/v4/class-rest-automation-workflows-controller.php

```diff
@@ -152,9 +152,22 @@ public function register_routes() {
 	 * @return WP_Error|WP_REST_Response
 	 */
 	public function get_items( $request ) {
+
+		// If $params['active'] is true then only active workflows should be returned.
```

Comment:
I changed this behavior because otherwise it is only possible to request either active workflows or inactive workflows, but not both together. The UI requires both together. Since the description of the `active` param is "Whether to return only active workflows", I thought this behavior seems to fit.